imageGenSD.py

```diff
@@ -82,6 +86,24 @@ def GetLoop(self):
 IGSD_client = IGSDClient(intents=intents)
 
 #####  Package Functions  #####
+def BannedWordsFound(prompt: str, banned_words: str) -> bool:
+    """Tests if banded words exist in the provided parameters.  This is written
+       as a separate function to allow future updates to the banned word list
+       without restarting the bot.
+
+       Input  : None.
+
+       Output : None.
+    """
+    result = False;
+    
+    #The default prompt is assumed to not be banned.
+    if banned_words != None and prompt != None:
+        #re-sanatize the word list in case spaces/separators snuck in
+        word_list = (banned_words.replace(" ", "")).split(",")
```

Comment:
Is the **replace** basically a strip? or do you want to get rid of spaces between words too?

Reply:
It fine

Reply:
Yeah, it's basically a strip.  I don't think I can remove all spaces and commas in a single strip command.